### ISSUE TITLE
Add share/subvol specific mount options

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#1826]
Thanks to forum member zanzida in the following forum thread for highlighting this issue. Having the ability to control share (subvol) mount options would server to enhance Rockstor's flexibility. This however should be done is a way that does not over complicate the UI. We already, as of pull request #1763, report share and pool mount options separately, but would have to extend the share model (and db) to encompass share specific mount options and deal with their interplay with such things as compression attribute settings vs mount compression settings.

Related forum post:
https://forum.rockstor.com/t/is-it-possible-to-specify-mount-options-for-individual-share/1709

**Comment by phillxnet** [comment]
Linking to very similar and prior entered issue by @modelrockettier #771

**Comment by rayfoss** [comment]
any workarounds? can be a serious issue in modern harddrives, qlc, an shingled drives

**Comment by rayfoss** [comment]
I'm thinking of testing out a manual remount sometime after boot...

```
/dev/sdg on /mnt2/big type btrfs (rw,relatime,space_cache,subvolid=258,subvol=/big) [main]
```

atime can make performance wildly unpredicatble

**Comment by lucserre** [comment]
I agree, atime or even relatime is very undesirable from a performance standpoint. Either shares (subvolumes) should copy pool mount options or have a section to manually adjust them.
